### PR TITLE
async function should stay on the same line

### PR DESCRIPTION
**Pull request by Xiot**
Deals with #630 

Sorry about the messy commit. It looks like the code got beautified and removed the tabs that were there.
I haven't figured out how to squash these commits into one yet, but if you really want I can go figure it out.

### CODE REVIEW

**Comment by Xiot** [comment]
I've never used python, so I don't know why the build is failing.

**Comment by bitwiseman** [comment]
Don't edit the test files directly.  Edit the 'test/data/javascript.js' file.

**Comment by bitwiseman** [comment]
Other than that you're change seems reasonable, but needs some more tests.

**Comment by Xiot** [comment]
No worries. I'll revert those files back and put the tests into the proper place.

**Comment by bitwiseman** [comment]
Those files will still end up being modified, but they'll be modified by the build. :)

**Comment by Xiot** [comment]
make testj doesnt seem to work on my machine (Windows 8 x64) it gives me 
```
$ make testj
Testing javascript implementation...
v0.12.0

> js-beautify@1.5.5 test D:\Development\Git Hub\js-beautify
> ./js/test/shell-smoke-test.sh

'.' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! Test failed.  See above for more details.
Makefile:39: recipe for target 'testj' failed
make: *** [testj] Error 1
```
However I was able to `make buildj` and run the js tests manually (`node .\js\test\node-beautify-tests.js`) and they all pass.

Python is still giving me an issue though
```
$ make buildp
Building python...
Obtaining file:///D:/Development/Git%20Hub/js-beautify/python
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "D:\Development\Git Hub\js-beautify\python\setup.py", line 4, in <mod                                                                          ule>
        from jsbeautifier.__version__ import __version__
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

      File "<string>", line 20, in <module>

      File "D:\Development\Git Hub\js-beautify\python\setup.py", line 4, in <mod                                                                          ule>

        from jsbeautifier.__version__ import __version__

    ----------------------------------------
    Command "python setup.py egg_info" failed with error code 1 in D:\Developmen                                                                          t\Git Hub\js-beautify\python
Makefile:28: recipe for target 'buildp' failed
make: *** [buildp] Error 1
```
Although I didn't touch that file at all.

File: python/jsbeautifier/__init__.py [hunk 1]
```diff
@@ -616,6 +616,10 @@ def handle_start_expr(self, current_token):
             # TODO: option space_before_conditional
             self.output.space_before_token = True
 
+        elif current_token.text == '(' && self.last_type == 'TK_RESERVED' && self.flags.last_word == 'await')
```
Comment (bitwiseman) [inline python/jsbeautifier/__init__.py]: Here's why the python is breaking.  You want `and` instead of `&&`.

**Comment by bitwiseman** [comment]
:+1:

**Comment by bitwiseman** [comment]
For this windows build error, could you file that as an issue?  We don't get much coverage on windows.

**Comment by Xiot** [comment]
woohoo !
I'll create an issue for the build errors on windows.

**Comment by bitwiseman** [comment]
I'm going to flatten this into a single commit and push the change directly.

**Comment by Xiot** [comment]
I haven't quite figured out how to do that yet. Time for some more learning.

**Comment by bitwiseman** [comment]
I've got it.  Good job on this and with the tests as well.

**Pull request closed without merging**